engine/src/main/scala/cromwell/engine/workflow/lifecycle/WorkflowExecutionActor.scala

```diff
@@ -45,32 +97,56 @@ object WorkflowExecutionActor {
   case object WorkflowExecutionAbortedResponse extends WorkflowExecutionActorResponse
   final case class WorkflowExecutionFailedResponse(reasons: Seq[Throwable]) extends WorkflowExecutionActorResponse
 
+  /**
+    * Captures the `ExecutionStoreKey`/`ExecutionStatus` combination for a call that has been created or started
```

Comment:
I think this comment needs an update. 
1. `JobKey / ExecutionStatus`
2. We can't only be using these for persistence if there isn't persistence in the actor yet?